src/odemis/util/img.py

```diff
@@ -777,3 +777,18 @@ def _getBoundingBox(content):
         md_pos[1] - half_shape_wc[1],
     )
     return rect
+
+
+def getOutliers(data, outliers=0):
+    """
+    Calculates the percentage of outliers that should be discarded.
+    :param data: (DataArray) The data containing the image.
+    :param outliers: (0<float<0.5) Ratio of outliers to discard (on both side). TODO does it discard 0.39 percent on each side??
```

Comment:
It discards 0.39% on both sides, if you specify 0.39/100. That docstring is quite clear about this, isn't it?